consensus/polybft/polybft.go

```diff
@@ -598,3 +599,38 @@ func (p *Polybft) GetBridgeProvider() consensus.BridgeDataProvider {
 func (p *Polybft) FilterExtra(extra []byte) ([]byte, error) {
 	return GetIbftExtraClean(extra)
 }
+
+func registerForksAndHandlers(forks *chain.Forks) error {
+	//nolint:godox
+	// TODO: update to real values and read from smart contract
+	availableForks := []forkmanager.ForkName{}
+	activeForks := []*forkmanager.ForkInfo{}
+	handlers := []forkmanager.ForkHandler{}
+
+	for name, block := range *forks {
+		if !chain.IsForkAvailable(name) {
+			panic(fmt.Errorf("fork is not available: %s", name)) //nolint:gocritic
+		}
+
+		availableForks = append(availableForks, forkmanager.ForkName(name))
+
+		if block != nil {
+			info := forkmanager.NewForkInfo(forkmanager.ForkName(name), (uint64)(*block))
+			activeForks = append(activeForks, info)
+		}
+	}
+
+	return forkmanager.GetInstance().RegisterAll(availableForks, handlers, activeForks)
+}
+
+func init() {
+	// init default fork
+	availableForks := []forkmanager.ForkName{""}
```

Comment:
Why do we need an empty fork?

Reply:
We can remove this for now... But lets say we introduce new handlers for some fork which starts from block N. In that case there is a need for handler from block 0

Reply:
removed https://github.com/0xPolygon/polygon-edge/pull/1544/commits/12b8f4149dd3f8517e875be1184833d0d94c1ef7